ui/packages/catalog-ui-search/src/main/webapp/component/confirmation/query/confirmation.query.hbs

```diff
@@ -13,12 +13,16 @@
  --}}
 <div class="confirmation-container">
     <div class="confirmation-prompt">
-        You are at the max number of searches allowed for a workspace.  
-        You can either select a current search to replace from the dropdown below,
-        or you can choose to start a new workspace with this search by not selecting 
-        any searches and simply pressing continue.
+        <div class="confirmation-header">
+            <span class="fa fa-exclamation-triangle"></span> Maximum Search Limit</span>
+        </div>
+        <hr>
+        You have reached the maximum number of searches (10) for the current workspace.
```

Comment:
✏️ Is the max number of searches configurable at all? If so, should we inject that config value into the hbs template?

Reply:
We hardcode the max number to 10.